prow/config/jobs/envoy.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
   - --token-path=/etc/github-token/oauth
   - --cmd=scripts/update_envoy.sh $AUTOMATOR_SHA $AUTOMATOR_SHA_COMMIT_DATE
   requirements: [github]
-  repos: [istio/test-infra@master]
+  repos: [istio/envoy@master, istio/test-infra@master]
```

Comment:
i wonder what's the `repos` field here means? I thought it is to find the repo which has the automator.sh? If we put two repos here, what it will do? Could you explain.

Reply:
They will be generated as the `extra_refs` field in the actual Prow job config, see https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md#what-the-test-container-can-expect for more information. Specifically
```
`periodic` jobs will have the working directory set to the root of the repo specified by the first ref in `extra_refs` (if specified). See the `extra_refs` field if you need to clone more than one repo.
```

Reply:
got it! that's might be why it checked out to test-infra SHA not envoy